packages/trader/src/App/Components/Routes/binary-link.jsx

```diff
@@ -16,11 +16,19 @@ const BinaryLink = ({ active_class, to, children, ...props }) => {
     }
 
     return to ? (
-        <NavLink to={path} activeClassName={active_class || 'active'} exact={route.exact} {...props}>
+        <NavLink
+            data-testid='dt_binary_link'
```

Comment:
do these have to be the same?
i mean this data-testid and the one which is in line 29

Reply:
Base on the conditions we can have the same testid for both of them